### ISSUE TITLE
Redirect loop attempting to load admin team pages

### ISSUE THREAD

**Issue by gregmarra** [the-blue-alliance/the-blue-alliance#2048]
## Expected Behavior
Expected: Try to load http://www.thebluealliance.com/admin/team/6024, page loads.

## Current Behavior
1. Try to load http://www.thebluealliance.com/admin/team/6024
2. Redirected to https://www.google.com/url?sa=D&q=http://www.thebluealliance.com/admin/team/6024&usg=AFQjCNHtJzBBWi3dfqQSMgNIemwGmXzknw
3. Page keeps redirecting to itself in a loop

## Context
Trying to delete some old social media accounts from a team, but cannot.

## Your Environment
Chrome 63.0.3239.84 (Official Build) (64-bit) on Mac OS

**Comment by fangeugene** [comment]
I'm not sure why this happens. I've seen it a couple of times and I think @phil-lopreiato has too. Revisiting the page later seems to resolve the problem.

**Comment by phil-lopreiato** [comment]
Yeah, I've seen this a few times too. Happy to know it's not a linux problem 😛

**Comment by phil-lopreiato** [comment]
I keep hitting this, but it doesn't happen when I go to https://tbatv-prod-hrd.appspot.com/admin/, so my hunch is that it has to do with the ssl config for tba.com? No idea where to go next, though
